src/settings.h

```diff
@@ -210,7 +210,8 @@
 	// (optinal) Neopixel
 	#ifdef NEOPIXEL_ENABLE
 		#define NUM_INDICATOR_LEDS		24          	// number of LEDs
-		#define NUM_STATIC_LEDS			0				// not used currently
+		#define NUM_CONTROL_LEDS		5		// number of control LEDs
```

Comment:
Please disable this feature by default (for existing users) and add some minimal hints, e.g.:

`#define NUM_INDICATOR_LEDS		24          	// number of Neopixel LEDs (formerly NUM_LEDS)
#define NUM_CONTROL_LEDS    	0				// optional control leds (https://forum.espuino.de/t/statische-ws2812-leds/1703)
#define CONTROL_LEDS_COLORS		{} 				// Colors for the control LEDs. Make sure it lists at least NUM_CONTROL_LEDS colors, e.g. for two control LEDs define: CONTROL_LEDS_COLORS {CRGB::Yellow, CRGB::Blue}
`

Sorry for bad formatting here..

![grafik](https://user-images.githubusercontent.com/11274319/236027573-8c4b4ff1-658d-42f9-b943-5d0066963776.png)

Reply:
Done. Also added a reference to the documentation for the available predefined CRGB colors and added a hint that plain hex codes could be used too.